src/main/java/org/broadinstitute/dsm/util/PatchUtil.java

```diff
@@ -104,7 +106,14 @@ private void getColumnNames(Class<?> obj) {
                     if (StringUtils.isNotBlank(columnPrefix)) {
                         fieldKey = columnPrefix.concat("_").concat(field.getName());
                     }
-                    columnNameMap.put(fieldKey, new DBElement(tableName, tableAlias, primaryKey, column.value()));
+                    DbDateConversion[] dbDateConversions = field.getAnnotationsByType(DbDateConversion.class);
+                    SqlDateConverter dateConverter = null;
+                    if (dbDateConversions != null && dbDateConversions.length == 1) {
```

Comment:
`getAnnotationsByType` anyway returns an empty array if there is no such annotation, so I think checking it on null is pointless, also since `DbDateConversion` is not `Repeatable`, using `getAnnotation` instead of `getAnnotationsByType` could be more convenient.